src/ipc_listener.cpp

```diff
@@ -193,7 +194,7 @@ void zmq::ipc_listener_t::in_event ()
 int zmq::ipc_listener_t::get_address (std::string &addr_)
 {
     struct sockaddr_storage ss;
-#ifdef ZMQ_HAVE_HPUX
+#if defined ZMQ_HAVE_HPUX
```

Comment:
Was this supposed to check if _VXWORKS is defined as well? If not please remove the change